include/librealsense2/dds/dds-device-impl.h

```diff
@@ -65,13 +66,17 @@ class dds_device::impl
     bool _running = false;
 
     size_t _num_of_sensors = 0;
-    std::unordered_map< int, std::string > _sensor_index_to_name;
-    std::unordered_map< int, topics::device::notification::video_stream_profiles_msg > _sensor_to_video_profiles;
-    std::unordered_map< int, topics::device::notification::motion_stream_profiles_msg > _sensor_to_motion_profiles;
-
-    eprosima::fastdds::dds::Subscriber* _subscriber;
-    eprosima::fastdds::dds::Topic* _topic;
-    eprosima::fastdds::dds::DataReader* _reader;
+    std::unordered_map< size_t, std::string> _sensor_index_to_name;
+    std::unordered_map< size_t, topics::device::notification::video_stream_profiles_msg > _sensor_to_video_profiles;
+    std::unordered_map< size_t, topics::device::notification::motion_stream_profiles_msg > _sensor_to_motion_profiles;
+    std::atomic<uint32_t> _message_counter = { 0 };
```

Comment:
This is the control outgoing message counter?

Reply:
Yes, I'll rename it `_control_message_counter`

Reply:
Done